Docker build fails - missing WguApp.vue

## Problem
After the merge of `merge 1.0.0-dev` into `master` the Docker build fails:
 
```

ERROR in ./src/main.js
Module not found: Error: Can't resolve '../app/WguApp' in '/app/src'
 @ ./src/main.js 5:0-35

  Build failed with errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wegue@1.0.0-dev build: `node build/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the wegue@1.0.0-dev build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
## Reason
The `npm` build expects `app/WguApp.vue` which is not present.

## Solution
Provide a `app/WguApp.vue` during the Docker build. Maybe create a `app/WguApp.vue.sample` file, that is renamed/copied into Docker image?

--- a/Dockerfile
+++ b/Dockerfile
@@ -6,7 +6,7 @@ WORKDIR /app
 COPY package*.json ./
 RUN npm install
 COPY . .
-RUN npm run build
+RUN npm run init:app && npm run build
 
 # production stage
 FROM nginx:stable-alpine as production-stage

